packages/passport-interface/src/FeedCredential.ts

```diff
@@ -0,0 +1,76 @@
+import { EmailPCD } from "@pcd/email-pcd";
+import { SerializedPCD } from "@pcd/pcd-types";
+import {
+  SemaphoreSignaturePCD,
+  SemaphoreSignaturePCDPackage
+} from "@pcd/semaphore-signature-pcd";
+
+// Timestamps last for one hour
+const TIMESTAMP_MAX_AGE = 1000 * 60 * 60;
+
+/*
+ * The payload encoded in the message of the SemaphoreSignaturePCD passed
+ * as a credential to feeds.
+ */
+export interface FeedCredentialPayload {
+  // The only type of PCD that can appear here is EmailPCD.
+  pcd?: SerializedPCD<EmailPCD>;
+  timestamp: number;
+}
+
+/**
+ * Creates a feed credential payload with timestamp.
+ *
+ */
+export function createFeedCredentialPayload(
+  pcd?: SerializedPCD
+): FeedCredentialPayload {
+  return {
+    pcd: pcd,
+    timestamp: Date.now()
```

Comment:
It looks like you're not aliasing timestamps (like the way we do it for frogs), and instead using the raw timestamp. Was that intentional? As far as I understand, this means we can't actually cache the verification result of this credential, because the client will generate a new one for each burst of requests (as you're re-instantiating the `CredentialManager` for each time you're making a request). I think not aliasing timestamps makes sense as a design decision - it just makes things complicated to a slightly uncomfortable degree.